package.json

```diff
@@ -50,11 +50,11 @@
     "react-dom": "^17.0.2",
     "react-modal": "^3.14.3",
     "react-popover": "CatchRelease/littlebits-react-popover#master",
-    "react-select": "^4.3.1",
-    "react-toastify": "^8.0.2",
+    "react-select": "^5.0.0",
```

Comment:
Also of note, `react-select` is now written in TypeScript, which may prove helpful when we convert our `AutoComplete`.